src/common/ecs-features/ecsFeatureClient.ts

```diff
@@ -33,10 +34,12 @@ export abstract class ECSFeaturesClient {
 
             // capture telemetry
             telemetry.sendTelemetryEvent(ECSConfigSuccessfulInit, { clientName: clientName, configFlagCount: Object.keys(this._ecsConfig).length.toString() });
```

Comment:
do we need this anymore then?

Reply:
Yes, for sending to older cluster. As per discussion with Bidisha that is not deprecated yet.